### PR TITLE
Bench the coq installation too

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

File: two_points_on_the_same_branch.sh [hunk 1]
```diff
@@ -192,10 +193,17 @@ create_opam() {
 
     echo "$1_coq_commit_long = $COQ_HASH_LONG"
 
-    if opam pin add -y -b -j$number_of_processors --kind=path coq.dev . ; then
-        echo "Coq installed successfully"
+    _RES=0
+    /usr/bin/time -o "$log_dir/coq.$RUNNER.1.time" --format="%U %M %F" \
+                  perf stat -e instructions:u,cycles:u -o "$log_dir/coq.$RUNNER.1.perf" \
+                  opam pin add -y -b -j "$number_of_processors" --kind=path coq.dev . \
```
Comment (SkySkimmer) [inline two_points_on_the_same_branch.sh:199]: It won't be super reliable especially with -j but better than nothing imo

**Comment by ejgallego** [comment]
Umm, not sure this won't be misleading, IMO a misleading bench may be worse than no bench. I remain very very wary of measuring OCaml compilation time as simple refactorings can have a large impact on compilation time.

Actually we could improve the setup a bit already, by splitting the `coq.opam` packages into `coq-base` and `coq-stdlib`, then we could measure `coq-stdlib`.

**Comment by SkySkimmer** [comment]
Let's worry about that once the split is done.

**Comment by ejgallego** [comment]
> Let's worry about that once the split is done.

OK; I'm still concerned about this bench, IMO it seems quite misleading and I'm worried that variations here will generate not very useful discussions, and in the end we will just ignore it.

But feel free to merge if you wanna have this.

p.s: Maybe we want to use a different development to test in Travis.

**Comment by ppedrot** [comment]
I personally prefer having a diff that is not reliable than no information, but YMMV.

**Comment by SkySkimmer** [comment]
If it's a problem we can always remove it.

**Pull request merged**